gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/tasks/explode/filter/entityfilters/StoredPasswordFilter.java

```diff
@@ -21,7 +21,8 @@
             PolicyFilter.class,
             ServiceFilter.class,
             JDBCConnectionFilter.class,
-            CassandraConnectionFilter.class).collect(Collectors.toSet());
+            CassandraConnectionFilter.class,
+            SsgActiveConnectorFilter.class).collect(Collectors.toSet());
```

Comment:
Can you also add `UnsupportedEntityFilter` and `GenericEntityFilter` to this list?

Reply:
ok